Makefile_i8_PaStiX

```diff
@@ -0,0 +1,133 @@
+# See our wiki for getting the CalculiX dependencies:
+# https://github.com/precice/calculix-adapter/wiki/Installation-instructions-for-CalculiX
+# Set the following variables before building:
+# Path to original CalculiX source (e.g. $(HOME)/ccx_2.16/src )
+#CCX             = $(HOME)/CalculiX/ccx_2.16/src
+CCX             = /usr/local/CalculiX/ccx_2.17/src
+
+### Change these if you built SPOOLES, ARPACK, or yaml-cpp from source ###
+# SPOOLES include flags (e.g. -I$(HOME)/SPOOLES.2.2 )
+SPOOLES_INCLUDE   = -I/usr/include/spooles/
+# SPOOLES library flags (e.g. $(HOME)/SPOOLES.2.2/spooles.a)
+SPOOLES_LIBS      = -lspooles
+#
+# ARPACK include flags (e.g. -I$(HOME)/ARPACK)
+ARPACK_INCLUDE    =
+# ARPACK library flags (e.g. $(HOME)/ARPACK/libarpack_INTEL.a)
+ARPACK_LIBS       = $(HOME)/ARPACK/libarpack_INTEL.a 
+#ARPACK_LIBS       = -larpack -llapack -lblas
+#
+# yaml-cpp include flags (e.g. -I$(HOME)/yaml-cpp/include)
+YAML_INCLUDE      = -I/usr/include/
+# yaml-cpp library flags (e.g. -L$(HOME)/yaml-cpp/build -lyaml-cpp)
+YAML_LIBS         = -lyaml-cpp
+
+# Get the CFLAGS and LIBS from pkg-config (preCICE version >= 1.4.0).
+# If pkg-config cannot find the libprecice.pc meta-file, you may need to set the
+# path where this is stored into PKG_CONFIG_PATH when building the adapter.
+PKGCONF_CFLAGS  = $(shell pkg-config --cflags libprecice)
+PKGCONF_LIBS    = $(shell pkg-config --libs libprecice)
+
+# Specify where to store the generated .o files
+OBJDIR = bin
+
+# For PaStiX only
+PASTIX_LIBS = $(HOME)/PaStiX/hwloc_i8/lib/libhwloc.so $(HOME)/OpenBLAS_i8/libopenblas.a \
+  $(HOME)/PaStiX/pastix_i8/lib/libpastix.a $(HOME)/OpenBLAS_i8/libopenblas.a -lpthread -lm $(HOME)/PaStiX/pastix_i8/lib/libspm.a \
+  $(HOME)/PaStiX/pastix_i8/lib/libpastix_parsec.a $(HOME)/PaStiX/pastix_i8/lib/libpastix_kernels.a $(HOME)/OpenBLAS_i8/libopenblas.a -lrt \
+  $(HOME)/PaStiX/pastix_i8/lib/libpastix_kernels_cuda.a $(HOME)/PaStiX/parsec_i8/lib/libparsec.so \
+  /usr/lib/x86_64-linux-gnu/libcudart_static.a -lpthread -ldl -lrt /usr/lib/x86_64-linux-gnu/libcublas_static.a \
+  /usr/lib/x86_64-linux-gnu/libculibos.a /usr/lib/x86_64-linux-gnu/libcublasLt_static.a -lrt \
+  $(HOME)/PaStiX/scotch_i8/lib/libscotch.a $(HOME)/PaStiX/scotch_i8/lib/libscotcherrexit.a -lpthread -lz -lm /usr/lib/x86_64-linux-gnu/libstdc++.so.6 /usr/lib/x86_64-linux-gnu/libgomp.so.1
```

Comment:
These paths sound very platform-specific. Maybe at least set this path prefix to a variable somewhere?

Reply:
What do you mean by prefix ?
I could split this into one variable per library though.

Reply:
By prefix, I mainly mean the `/usr/lib/x86_64-linux-gnu/`.